src/components/App.js

```diff
@@ -1,28 +1,23 @@
 'use strict'
 
 import React from 'react'
+import createReactClass from 'create-react-class'
+import PropTypes from 'prop-types'
 import { render } from 'react-dom'
 import { Router, Route, hashHistory } from 'react-router'
 import Logger from 'logplease'
 
 import AppActions from 'actions/AppActions'
-import UIActions from "actions/UIActions"
+import UIActions from 'actions/UIActions'
 import NetworkActions from 'actions/NetworkActions'
-import NotificationActions from 'actions/NotificationActions'
 import IpfsDaemonActions from 'actions/IpfsDaemonActions'
-import ChannelActions from 'actions/ChannelActions'
 
 import OrbitStore from 'stores/OrbitStore'
-import IpfsDaemonStore from 'stores/IpfsDaemonStore'
 import AppStateStore from 'stores/AppStateStore'
 import UserStore from 'stores/UserStore'
-import UserActions from 'actions/UserActions'
 import NetworkStore from 'stores/NetworkStore'
 import ChannelStore from 'stores/ChannelStore'
-import MessageStore from 'stores/MessageStore'
-import UsersStore from 'stores/UsersStore'
 import SettingsStore from 'stores/SettingsStore'
-import SwarmStore from 'stores/SwarmStore'
```

Comment:
Hah, awesome to remove so many unnecessary(?) deps from the component! 👍

Reply:
Yup they were unused :)